src/xpring-client.ts

```diff
@@ -19,10 +18,23 @@ class XpringClient {
    * @param grpcURL The URL of the gRPC instance to connect to.
    * @param useNewProtocolBuffers If `true`, then the new protocol buffer implementation from rippled will be used. Defaults to false.
    */
-  public constructor(grpcURL: string, useNewProtocolBuffers = false) {
-    const defaultXpringClient: XpringClientDecorator = useNewProtocolBuffers
-      ? DefaultXpringClient.defaultXpringClientWithEndpoint(grpcURL)
-      : LegacyDefaultXpringClient.defaultXpringClientWithEndpoint(grpcURL)
+  public constructor(
+    grpcURL: string,
+    useNewProtocolBuffers = false,
```

Comment:
doc parameter

I think ternary is cleaner than if/else syntax, do you not like it?

Reply:
done